src/models/ServerStyleSheet.js

```diff
@@ -0,0 +1,8 @@
+import styleSheet from './StyleSheet'
+
+styleSheet.reset = styleSheet.flush
```

Comment:
Hmm, I'd prefer not mutating things here. I'd suggest this:

```js
class ServerStyleSheet {
  reset() {
    styleSheet.flush()
  }
  getCSS() {
    return styleSheet.rules().map(rule => rule.cssText).join('\n')
  }
}

export default new ServerStyleSheet()
```

That way if `styleSheet` changes we don't have potentially public-facing changes to worry about here.